src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -380,7 +380,9 @@ private static DynamicJsonValue GetSubscriptionConnectionDJV(SubscriptionConnect
                 [nameof(SubscriptionConnection.ClientUri)] = x.ClientUri,
                 [nameof(SubscriptionConnection.Strategy)] = x.Strategy,
                 [nameof(SubscriptionConnection.Stats)] = GetConnectionStatsDJV(x.Stats),
-                [nameof(SubscriptionConnection.ConnectionException)] = x.ConnectionException?.Message
+                [nameof(SubscriptionConnection.ConnectionException)] = x.ConnectionException?.Message,
+                ["TCPConnectionStats"] = x.TcpConnection.GetConnectionStats(),
```

Comment:
`nameof` ?

Reply:
I think that in that case, especially because it's a debug endpoint, it's more important to maintain clearer function name then to maintain "strongly typed" structure

Reply:
For consistency with other endpoints, this should be `Tcp` instead of `TCP`